Disable adding -march=native on clang architectures. This allows use
of instructions that are present on the build machine but not on other
CPU types within the given architecture, so is not suitable for package
builds.

Index: configure
--- configure.orig
+++ configure
@@ -19245,7 +19245,7 @@ printf "%s\n" "$as_me: WARNING: LAME doesn't know abou
 	esac
 
 
-	if test "x${HAVE_CLANG}" = "xyes"; then
+	if false && test "x${HAVE_CLANG}" = "xyes"; then
 		case "${CLANG_VERSION}" in
 		3.[89]*|[45].*|1[0-9].*)
 			OPTIMIZATION="-Ofast"
